### ISSUE TITLE
Form error messages are not announced by screen reader

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-accessibility-passport#119]
Although validations are not ready to fully test against this criterion, the current situation is that the error messages are not announced by the screen reader.

### Screenshot
![image.png](https://images.zenhubusercontent.com/5f7b33373ac7b7279de9a3c5/1b83556c-29fc-48bf-a63a-896dcdbcfce6)

**Comment by mnigh** [comment]
Duplicate of #111.
